email_reply_parser/__init__.py

```diff
@@ -282,6 +282,14 @@ def clean_email_content(body, include: Optional[bool], word_limit):
             # Remove the sign-off
             body = EmailMessage.remove_signoff(body, signoff_matches, word_limit)
 
+        # Split the body into lines and remove the last line if it matches any of the SIG_REGEX
+        # Use the unidecode copy to check for matches but remove lines from original body
+        body_lines_unidecode = body_clean.split('\n')
+        body_lines = body.split('\n')
```

Comment:
Is there a chance here that `body_lines_unidecode` and `body_lines` won't be the same after `body` is parsed in the lines above?

Maybe this should be something like:

```python
body = body.strip()

body_lines = body.split('\n')
body_lines_unidecode = unidecode.unidecode(body).split('\n')
```